### DISCUSSION TITLE
need help with setting up LRCGET scoop package

### DISCUSSION THREAD

**Discussion by back-lacking** [ScoopInstaller/Scoop#6226] [Help and Questions]
exactly what the title says
im trying to make a package for [LRCGET](https://github.com/tranxuanthang/lrcget) an app for creating and/or pulling synced lyrics files for/from lrclib.net. 
ive tried looking the documentation for how to make a scoop package and looked at other packages using the `.msi` package format but so far ive failed and don't really know what to do atm. im debating on whether or  not i should move over to the `.exe` installer, considering the file structure of it isnt really complex. it just contains the app itself, the uninstaller, and an `$PLUGINSDIR` folder containing a few dlls. [json file here](https://github.com/back-lacking/personal-scoop/blob/master/bucket/lrcget.json)
im testing the package install in an win11 vm as i dont want to mess up my current win10 install. here is the error i get in my vm
![image](https://github.com/user-attachments/assets/9e263626-6a68-4b64-b83e-07133f2c128d)

**Comment by HUMORCE** [comment]
Many manifest for the app already created by 3rd-party buckets: https://scoop.sh/#/apps?q=LRCGET&dm=false

Additionally, the same type of the manifest(in the case, the type is MSI extraction) exists in at least a hundred on official buckets.

https://github.com/search?q=repo%3AScoopInstaller%2FExtras++OR+repo%3AScoopInstaller%2FMain+.msi&type=code


```
{
    "version": "0.5.0",
    "description": "Utility for mass-downloading LRC synced lyrics for your offline music library. ",
    "homepage": "https://github.com/tranxuanthang/lrcget",
    "license": "MIT",
    "url": "https://github.com/tranxuanthang/lrcget/releases/download/0.5.0/LRCGET_0.5.0_x64_en-US.msi",
    "hash": "4CF6D43854AA26C223D146D86AFBB8FBEDB122246352B2734097E1336E5C816D",
### MODIFY START
#   - You can view the inner directory structure with `main/lessmsi`
#   - The behavior of msiexec.exe and lessmsi is inconsistent in some cases.
#   - Remove this property first then test it step by step.
    "extract_dir": "PFiles\\LRCGET",
### MODIFY END
    "bin": "LRCGET.exe",
### REMOVE START
#   - Uninstaller is unnecessary, because you just extract files from installer instead execute it.
    "uninstaller": {
        "file": "uninstall.exe"
    },
### REMOVE END
    "shortcuts": [
        [
            "LRCGET.exe",
            "LRCGET"
        ]
    ],
    "checkver": "github",
    "autoupdate": {
        "url": "https://github.com/coreybutler/nvm-windows/releases/download/$version/LRCGET_$version_x64_en-US.msi"
    }
}
```
